sdk/tables/azure-data-tables/src/transactions.cpp

```diff
@@ -1,8 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-#include <azure/data/tables/internal/serializers.hpp>
-#include <azure/data/tables/transactions.hpp>
+#include "azure/data/tables/transactions.hpp"
+

```

Comment:
nit: new line not needed.

Just to understand, are you doing it to separate the internal header includes from the public header ones?

Reply:
new line was added by clang , ask Larry , it separates by folders and <>